pandas/io/common.py

```diff
@@ -161,6 +176,7 @@ def is_fsspec_url(url: FilePathOrBuffer) -> bool:
     return (
         isinstance(url, str)
         and "://" in url
+        and not is_json(url)
```

Comment:
For pandas maintainers: should this check only be done within read_json (which will presumably call json.loads anyway, repeating the work) ? Are there any other similar cases that the string passed might be handled as a path, or decoded directly, that I might have missed?

Reply:
`read_json` doesn't call `json.loads`

Reply:
@tbachlechner if you really think t his out to be caught then i suppose a 'better' check instead of ```and "://" in url``` could be done (e.g. a regex)